node/src/components/fetcher.rs

```diff
@@ -154,53 +161,73 @@ pub trait ItemFetcher<T: Item + 'static> {
         }
     }
 
+    /// Sends result to all responders
+    fn respond_to_all(&mut self, id: T::Id, result: FetchResult<T, NodeId>) -> Effects<Event<T>> {
```

Comment:
So this is never meant to be called with an `Err` result? Would be good to document that.

Reply:
Fixed in 9fedf1eef373bce79649cb8849b461b95515f386